cmd/publishing-bot/log_builder.go

```diff
@@ -101,6 +108,20 @@ func (builder *logBuilder) Join(sep string) *logBuilder {
 	return builder
 }
 
+func (builder *logBuilder) Unique() *logBuilder {
```

Comment:
why do we need this?

Reply:
i add this for the test case, does that make sense?